main.py

```diff
@@ -2,23 +2,90 @@
 #
 # Simple manager prototype for xqemu
 #
-from PyQt5.QtWidgets import QApplication, QDialog, QFileDialog, QMainWindow, QMessageBox
+from PyQt5.QtWidgets import QApplication, QDialog, QFileDialog, QMainWindow, QMessageBox, QTableWidgetItem, QAbstractItemView, QHeaderView
 from PyQt5.uic import loadUiType
 from PyQt5 import QtCore, QtGui
 from qmp import QEMUMonitorProtocol
+from glob import glob1
 import sys
 import os, os.path
 import json
 import subprocess
 import time
 import platform
+import csv
+import sys
 
 SETTINGS_FILE = './settings.json'
+LIBRARY_CACHE = './library.cache'
```

Comment:
Still reviewing, so not sure how this is used yet, but consider using json for this, too.